pkg/cmd/application/create/app_create.go

```diff
@@ -209,7 +233,7 @@ GitHub CLI commands:
   gh workflow run <workflow-id> -R %s
 `
 	nextStepsMessageTemplateWithoutPR = `
-Note: to complete application onboarding to the Core Platform you have to first merge PR that adds this application to your repository.
+To complete application onboarding to the Core Platform you have to first merge PR that adds this application to your repository.
 PR url: %s
```

Comment:
Not you part of the PR, but it looks like the messages are the same. We should update it (here or in another task)

Reply:
They are slightly different, the steps look correct to me (I've not tried a monorepo myself yet!)

Reply:
Ah, right, sorry